bin/common/create_kubedns.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "Waiting for kubectl to respond..."
+while ! kubectl get pods --all-namespaces; do
+  sleep 2
+done
+echo "Waiting for kube-apiserver to be active..."
+while ! systemctl is-active kube-apiserver.service 2>/dev/null >/dev/null; do
```

Comment:
I must have accidentally swapped the order here from https://github.com/SUSE/scf/blob/develop/packer/Vagrantfile.template#L75-L82 .  Nice catch!